src/daemon.py

```diff
@@ -30,100 +29,74 @@ def compile_yara(job_hash: str) -> Any:
     return rule
 
 
-def get_list_name(priority: str) -> str:
-    if priority == "low":
-        return "list-yara-low"
-    elif priority == "medium":
-        return "list-yara-medium"
-    else:
-        return "list-yara-high"
-
-
 def collect_expired_jobs() -> None:
     if config.JOB_EXPIRATION_MINUTES <= 0:
         return
 
     exp_time = int(60 * config.JOB_EXPIRATION_MINUTES)  # conversion to seconds
-    job_hashes = []
+    job_ids: List[JobId] = []
 
-    for job_hash in redis.keys("job:*"):
-        job_hashes.append(job_hash[4:])
+    for job in redis.get_job_ids():
+        job_ids.append(job)
 
-    for job in job_hashes:
-        redis.set("gc-lock", "locked", ex=60)
-        job_submitted_time = int(redis.hget("job:" + job, "submitted"))
+    for job in job_ids:
+        job_submitted_time = redis.get_job_submitted(job)
```

Comment:
Nitpicking but perhaps `job_submission_time`?

Reply:
English is hard